MDC2020/gen_Mix.sh

```diff
@@ -66,8 +66,9 @@ if [ $1 == "NoPrimary" ]; then
 elif [ $1 == "NoPrimaryPBISequence" ]; then
   echo '#include "Production/JobConfig/mixing/NoPrimaryPBISequence.fcl"' >> template.fcl
 else
+  samListLocations --defname="dts.mu2e.$1.$2$4.art" > $1$4.txt
```

Comment:
I was going to do this, but since you are modifying the script, can you change `samListLocations` entries to:
```
samweb list-file-locations --defname=<dataset_name> --schema=root | cut -f1
```
This writes the file paths as xrootd URL